main.go

```diff
@@ -97,15 +105,31 @@ func main() {
 	service.Use(middleware.ErrorHandler(service, true))
 	service.Use(middleware.Recover())
 
```

Comment:
@aslakknutsen @baijum : Should we consider adding a custom simple middleware, that will set `request.currentuser` for every coming request ? For non-auth requests it should set `request.currentUser=nil`.
This will help to identify the user in the request handler.